### PR TITLE
⚡️ Comment embeds

### PR DESCRIPTION
**Pull request by nicl**
See also https://github.com/guardian/frontend/pull/20755 which needs to go live first.

Adds comment element styling:

screenshot 2018-11-22 at 16 54 29

### CODE REVIEW

File: frontend/lib/parse-capi/index.ts [hunk 1]
```diff
@@ -345,8 +349,7 @@ export const extractArticleMeta = (data: {}): CAPIType => {
         ),
         pageId: getNonEmptyString(data, 'config.page.pageId'),
         sharingUrls: getSharingUrls(data),
-        pillar:
-            findPillar(getNonEmptyString(data, 'config.page.pillar')) || 'news',
+        pillar: findPillar(getString(data, 'config.page.pillar', '')) || 'news',
```
Comment (AWare) [inline frontend/lib/parse-capi/index.ts]: Should ` || 'news'` go here or in `findPillar`?

**Comment by AWare** [comment]
👍 There are some _weird_ merge conflicts going on here.

**Comment by SiAdcock** [comment]
> _weird_ merge conflicts

They are my fault and I will fix. Sorry

File: frontend/lib/parse-capi/index.ts (hunk 1 above)
Comment (nicl) [reply frontend/lib/parse-capi/index.ts]: that feels like a much better approach :)

**Comment by nicl** [comment]
@SiAdcock it's okay. Turns out it merged without any conflicts! 🏅 🎉

File: frontend/lib/parse-capi/index.ts (hunk 1 above)
Comment (nicl) [reply frontend/lib/parse-capi/index.ts]: In fact, findPillar will always return a `Pillar | undefined` so keeping  things as they are probably makes sense here after all.

**Pull request merged**